sys/usb/usbus/cdc/acm/cdc_acm_stdio.c

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) 2018 Koen Zandberg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup sys
+ * @{
+ *
+ * @file
+ * @brief CDC ACM stdio implementation for USBUS CDC ACM
+ *
+ * This file implements a USB CDC ACM callback and read/write functions.
+ *
+ *
+ * @}
+ */
+
+#include <stdio.h>
+
+#include "isrpipe.h"
+
+#include "usb/usbus.h"
+#include "usb/usbus/cdc/acm.h"
+
+#if MODULE_VFS
+#include "vfs.h"
+#endif
+
+static usbus_cdcacm_device_t cdcacm;
```

Comment:
Is it possible to remove the static keyword so the `cdcacm` struct can be reach from outside using `extern` ?

Reply:
You probably have some kind of goal in mind with this, care to clarify what you're trying to create. Furthermore do we want to stall this PR to ensure a simple follow up or is it okay to rework things slightly later?

Reply:
accessing this using `extern` feels like a small & ugly hack ;)

Reply:
It is! But if you have a better idea for calling `usbus_cdc_acm_set_coding_cb(&cdcacm, _coding_cb);` from `board_init`, I buy it :)

Reply:
What do you want to achieve by calling this from `board_init` ?

Reply:
How about adding something like
```C
usbus_cdcacm_device_t * usb_cdc_acm_stdio_get_ctx(void) 
{
    return &cdcacm;
}
```

This assumes that there is always at most one stdio (which imho is not a bad assumption), and we can change the implementation if required in the future.

Reply:
> How about adding something like [...]

That might be cleanest. I tried something else using pseudomodules and conditional macros but I always end up having to `extern` something (a function in my second attempt).

Anyway, let's not change this in this PR. I can adapt this module like you suggest when I have something ready (I have already but I have to test it first).

Reply:
> Anyway, let's not change this in this PR. I can adapt this module like you suggest when I have something ready (I have already but I have to test it first).

Thanks!